gui/wxpython/lmgr/frame.py

```diff
@@ -1302,20 +1302,6 @@ def _popupMenu(self, data):
         self.PopupMenu(menu)
         menu.Destroy()
 
-    def OnImportMenu(self, event):
-        """Import maps menu (import, link)
-        """
-        self._popupMenu((('rastImport', self.OnImportGdalLayers),
-                         ('vectImport', self.OnImportOgrLayers),
-                         (None, None),
-                         ('rastUnpack', self.OnUnpackRaster),
-                         ('vectUnpack', self.OnUnpackVector),
-                         (None, None),
-                         ('rastLink', self.OnLinkGdalLayers),
-                         ('vectLink', self.OnLinkOgrLayers),
-                         ('rastOut', self.OnRasterOutputFormat),
-                         ('vectOut', self.OnVectorOutputFormat)))
-
```

Comment:
Some of these methods are not used anymore so they should be removed.

Reply:
They are used in RunSpecialCmd function in lgmr/frame.py. Not sure if they can be removed from "elif" parts in RunSpecialCmd.

Reply:
Not the unpack ones.